src/libraries/System.Numerics.Tensors/src/System/Numerics/Tensors/netcore/TensorPrimitives.netcore.cs

```diff
@@ -1709,32 +1979,338 @@ static T Vectorized512(ref T xRef, ref T yRef, nuint remainder)
                                                                      Vector512.LoadUnsafe(ref yRef, remainder - (uint)(Vector512<T>.Count * 1)));
                         vresult = TAggregationOperator.Invoke(vresult, vector);
                         goto case 0;
-                    }
+                    }
+
+                    case 0:
+                    {
+                        // Store the last block, which includes any elements that wouldn't fill a full vector
+                        end = Vector512.ConditionalSelect(CreateRemainderMaskVector512<T>((int)trailing), end, Vector512.Create(TAggregationOperator.IdentityValue));
+                        vresult = TAggregationOperator.Invoke(vresult, end);
+                        break;
+                    }
+                }
+
+                return TAggregationOperator.Invoke(vresult);
+            }
```

Comment:
git chose an interesting way to do the diff here, this isn't new, it looks identical to the previous code.